portfolio/src/main/java/com/google/DataServlet.java

```diff
@@ -24,30 +24,34 @@
  
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data") 
-public class DataServlet extends HttpServlet {
-
-  private List<String> phrases;
+public class DataServlet extends HttpServlet 
+{ 
+  private List<String> comments;
 
    @Override
   public void init() {
-    phrases = new ArrayList<String>();
-    phrases.add("Writer");
-    phrases.add("Achiever");
-    phrases.add("Engineer");
-    phrases.add("Student");
-    phrases.add("Visionary");
+    comments = new ArrayList<String>();
   }
 
    @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJsonUsingGson(phrases);
+    String json = convertToJsonUsingGson(comments);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+
+   @Override
```

Comment:
nit: think there is an extra space before `@Override`.

Reply:
I placed only one space between each of the fields and methods of the DataServlet file

Reply:
Oh I see which space you were talking about, I'll fix that now